apps/hubspot/functions/createModules.ts

```diff
@@ -76,3 +90,84 @@ const createModuleFile = async (
     );
   }
 };
+
+const getFields = (field: SdkField): string => {
+  const { type } = field;
+  let fields;
+  switch (type) {
+    case 'Symbol':
+      fields = structuredClone(TEXT_FIELD_TEMPLATE);
```

Comment:
💭 I feel this could be in the createModule, because we already are calculating the module in line 59. Something like
```suggestion
      fields = structuredClone(module);
```